src/db.rs

```diff
@@ -71,6 +71,59 @@ impl DB {
         Ok(())
     }
 
+    pub fn migrate<P: AsRef<Path>>(&mut self, path: P) -> Result<()> {
+        if !self.dirs.is_empty() {
+            return Err(anyhow!(
+                "To prevent conflicts, you can only migrate from z with an empty \
+                 zoxide database!"
+            ));
+        }
+
+        let zdata = File::open(path)?;
```

Comment:
Add `with_context` here, so that the user can see what exactly caused the error. Also, a more descriptive name would be nice, like `z_file`.

Reply:
`z_db` or `z_database`, maybe? `z_file` seems even less descriptive than `zdata`, IMHO.

Reply:
Maybe we could meet in the middle and do `z_dbfile` or `z_db_file`? ;)

Reply:
`z_db_file` sounds fine!